docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -113,4 +134,10 @@ The equivalent XML configuration is as follows:
 include::{sourcedir}/clustered/client/src/test/resources/configs/docs/ehcache-clustered.xml[tag=consistencySample]
 ----
 
-<1> Specify the consistency level through a custom service configuration from the `clustered` namespace.
\ No newline at end of file
+<1> Specify the consistency level through a custom service configuration from the `clustered` namespace.
+
+=== Clustered Cache Expiry
+
+Expiry in clustered caches work with an exception that `Expiry#getExpiryForAccess` is ignored in clustered tier, though the same
```

Comment:
I don't think we want to be blunt about the fact that it is ignored. I think best effort would be the right terms to use to disguise the reality. @ljacomet WDYT?